### ISSUE TITLE
Cursor rect doesn't change when in dark mode.

### ISSUE THREAD

**Issue by ryanbugden** [typemytype/shapeToolRoboFontExtension#5]
Maybe it should be white? Or match font dimensions color pref?

**Comment by typemytype** [comment]
yeah it could maybe use https://robofont.com/RF4.4/documentation/reference/api/mojo/mojo-ui/#mojo.UI.appearanceColorKey 

now its red and blue... with the idea that this is a color that both matches light and dark mode
